CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Categories for each release: Added, Changed, Deprecated, Removed, Fixed, Securit
 ### Added
 
 * support for `--destination` in `nextflow build --repository`
+* add `resume` and `preserve_cache` input arguments to nextflow applet to support Nextflow resume functionality
```

Comment:
@commandlinegirl I added this here but feel free to change or remove it if resume shall not be 'visible' now

Reply:
LG!